application/tests/api/EmailCest.php

```diff
@@ -0,0 +1,190 @@
+            'subject' => 'subject allowed fields',
+            'text_body' => 'text body',
+            'html_body' => 'html body',
+        ]);
+        $I->seeResponseCodeIs(200);
+    }
+
+    public function testQueue_AllFields(ApiTester $I)
+    {
+        $I->wantTo('queue an email with all fields');
+        $I->haveHttpHeader('Authorization', 'Bearer abc123');
+        $I->sendPOST('/email', [
+            'id' => 123,
+            'to_address' => 'test@example.org',
+            'cc_address' => 'testcc@example.org',
+            'bcc_address' => 'testbcc@example.org',
+            'subject' => 'subject all fields',
+            'text_body' => 'text body',
+            'html_body' => 'html body',
+            'attempts_count' => 456,
+            'created_at' => 11111111,
+            'updated_at' => 22222222,
+            'error' => 'error message',
+        ]);
+        $I->seeResponseCodeIs(200);
+        $I->dontSeeResponseContains('123');
+        $I->dontSeeResponseContains('456');
+        $I->dontSeeResponseContains('11111111');
+        $I->dontSeeResponseContains('22222222');
+        $I->dontSeeResponseContains('error message');
+    }
+
+    public function testQueue_NoAccessToken(ApiTester $I)
+    {
+        $I->wantTo('queue an email without a bearer access token');
+        $I->sendPOST('/email');
+        $I->seeResponseCodeIs(401);
+    }
+
+    public function testQueue_InvalidAccessToken(ApiTester $I)
+    {
+        $I->wantTo('queue an email with an invalid bearer access token');
+        $I->haveHttpHeader('Authorization', 'Bearer invalid');
+        $I->sendPOST('/email');
+        $I->seeResponseCodeIs(401);
+    }
+
+    public function testInvalidPath(ApiTester $I)
+    {
+        $I->wantTo('queue an email using the wrong path');
+        $I->haveHttpHeader('Authorization', 'Bearer abc123');
+        $I->sendPOST('/invalid');
+        $I->seeResponseCodeIs(404);
+    }
+
+    public function testInvalidMethodGet(ApiTester $I)
+    {
+        $I->wantTo('queue an email using a GET');
+        $I->haveHttpHeader('Authorization', 'Bearer abc123');
+        $I->sendGet('/email');
+        $I->seeResponseCodeIs(404);
+    }
+
+    public function testInvalidMethodDelete(ApiTester $I)
+    {
+        $I->wantTo('queue an email using a DELETE');
+        $I->haveHttpHeader('Authorization', 'Bearer abc123');
+        $I->sendDELETE('/email');
+        $I->seeResponseCodeIs(404);
+    }
+
+    public function testInvalidMethodPut(ApiTester $I)
+    {
+        $I->wantTo('queue an email using a PUT');
+        $I->haveHttpHeader('Authorization', 'Bearer abc123');
+        $I->sendPUT('/email');
+        $I->seeResponseCodeIs(404);
+    }
+
+    public function testQueue_RequiredFieldsMissing_ToAddress(ApiTester $I)
+    {
+        $I->wantTo('queue an email without the required to_address');
+        $I->haveHttpHeader('Authorization', 'Bearer abc123');
+        $I->sendPOST('/email', [
+        ]);
```

Comment:
Long-term, it may be worth adding the other required fields here, since this test would pass even if `to` wasn't a required field (since a `422` would be returned because of other missing required fields).

Similarly for the other tests for missing required fields (only leaving out the required field being tested would improve the tests).